frontend/javascripts/oxalis/controller/scene_controller.js

```diff
@@ -321,27 +321,22 @@ class SceneController {
     // all buckets necessary for rendering are addressed. The anchorPoint is
     // defined with bucket indices for the coordinate system of the current zoomStep.
     let anchorPoint;
-    // The fallbackAnchorPoint is similar to the anchorPoint, but refers to the
-    // coordinate system of the next zoomStep which is used for fallback rendering.
-    let fallbackAnchorPoint;
 
     const zoomStep = getRequestLogZoomStep(Store.getState());
     for (const dataLayer of Model.getAllLayers()) {
-      [anchorPoint, fallbackAnchorPoint] = dataLayer.layerRenderingManager.updateDataTextures(
-        globalPosition,
-        zoomStep,
-      );
+      anchorPoint = dataLayer.layerRenderingManager.updateDataTextures(globalPosition, zoomStep);
     }
 
     if (optArbitraryPlane) {
-      optArbitraryPlane.updateAnchorPoints(anchorPoint, fallbackAnchorPoint);
+      optArbitraryPlane.updateAnchorPoints(anchorPoint);
       optArbitraryPlane.setPosition(globalPosVec);
     } else {
       for (const currentPlane of _.values(this.planes)) {
-        currentPlane.updateAnchorPoints(anchorPoint, fallbackAnchorPoint);
+        currentPlane.updateAnchorPoints(anchorPoint);
         currentPlane.setPosition(globalPosVec);
         const [scaleX, scaleY] = getPlaneScalingFactor(state, flycam, currentPlane.planeID);
         currentPlane.setScale(scaleX, scaleY);
+        this.displayPlane[currentPlane.planeID] = scaleX > 0 && scaleY > 0;
```

Comment:
> So planes in the TDview are now hidden, if the tab containing that plane is scaled to be very very small?

Almost :) If the viewport is hidden (happens if the tab is not active), then the scale is 0 and thus, the plane is hidden. This didn't came with this PR, but was there before. However, the problem is that threeJS complains since it tries to invert some matrix for which the determinant is zero (due to the scale being 0). This line is supposed to suppress the warning. However, I just noticed that it didn't work properly (scale needs to be non-zero, too). I just fixed that.

> Would you say users expect this or what is the reasoning behind that?

I think, it's alright since nobody has complained yet. Also, I think it's plausible, since hiding a viewport means that you are not really interested in it. So, hiding it in the 3d viewport as well, makes sense to me. Additionally, it's beneficial to perf :)